src/components/new-safe/CreateSafe/useSetCreationStep.ts

```diff
@@ -4,20 +4,22 @@ import type { StepRenderProps } from '@/components/new-safe/CardStepper/useCardS
 import type { PendingSafeData } from '@/components/new-safe/steps/Step4'
 import type { NewSafeFormData } from '@/components/new-safe/CreateSafe/index'
 import { SAFE_PENDING_CREATION_STORAGE_KEY } from '@/components/new-safe/steps/Step4'
+import useWallet from '@/hooks/wallets/useWallet'
 
-const useSetCreationStep = (setStep: StepRenderProps<NewSafeFormData>['setStep'], isConnected: boolean) => {
+const useSetCreationStep = (setStep: StepRenderProps<NewSafeFormData>['setStep']) => {
```

Comment:
Nit: can we rename this to be a bit more declarative, e.g. `useSyncSafeCreationStep`.